inverse/ft_inverse_eloreta.m

```diff
@@ -133,37 +133,37 @@
   end
 end
 
-% deal with reduced rank
-% check the rank of the leadfields, and project onto the lower dimensional
-% subspace if the number of columns per leadfield > rank: this to avoid
-% numerical issues in the filter computation
-rank_lf = zeros(1,size(sourcemodel.pos,1));
-for i=1:size(sourcemodel.pos,1)
-  rank_lf(i) = rank(sourcemodel.leadfield{i});
-end
-if ~all(rank_lf==rank_lf(1))
-  ft_error('the forward solutions have a different rank for each location, which is not supported');
-end
-if rank_lf(1)<size(sourcemodel.leadfield{1})
-  ft_notice('the forward solutions have a rank of %d, but %d orientations\n',rank_lf(1),size(sourcemodel.leadfield{1},2));
-  ft_notice('projecting the forward solutions on the lower dimensional subspace\n');
-  for i=1:size(sourcemodel.pos,1)
-    [u,s,v{i}] = svd(sourcemodel.leadfield{i}, 'econ');
-    sourcemodel.leadfield{i} = sourcemodel.leadfield{i}*v{i}(:,1:rank_lf(i));
-  end
-end
-
-% convert the leadfield into Nchan*Ndip*Nori
-[Nchan, Nori] = size(sourcemodel.leadfield{1});
-Ndip          = numel(sourcemodel.leadfield);
-leadfield     = permute(reshape(cat(2,sourcemodel.leadfield{:}),Nchan,Nori,Ndip),[1 3 2]);
-
 % use existing filters, or compute them
 if ~hasfilter
-  filt = mkfilt_eloreta(leadfield, lambda);
-  for i=1:size(sourcemodel.pos,1)
-    sourcemodel.filter{i,1} = squeeze(filt(:,i,:))';
-  end
+    % deal with reduced rank
```

Comment:
It's unclear what the change in the code is here, the largest diff is due to indentation changing from the (conventional) 2, to 4.

Reply:
Indentation changing is due to refractoring the code. The main issue was that the function was looking for the leadfield even when a precomputed filter was provided. This commit puts the leadfield processing part under the condition check `~hasFilter`.

Reply:
Well, I changed some of the indentation, because the tab length was also changed.